source/uri-options/tests/concern-options.json

```diff
@@ -36,15 +36,6 @@
         "w": "arbitraryButStillValid"
       }
     },
-    {
```

Comment:
It looks to me like this test [exists in YML](https://github.com/mongodb/specifications/blob/master/source/uri-options/tests/concern-options.yml#L41), why is it being deleted here?

Reply:
The test you are linking to is a different test which exists in the json.

Reply:
I'm commenting on removed JSON from `source/uri-options/tests/concern-options.json`, and I'm linking to `source/uri-options/tests/concern-options.yml`. How is that a different test? And where is that test rendered to JSON?

Reply:
oof, sorry the tests are so similarly named.